controllers/clustercurator_controller.go

```diff
@@ -42,6 +44,16 @@ func (r *ClusterCuratorReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, nil
 	}
 
+	if curator.Spec.DesiredCuration == DeleteNamespace {
+		log.V(0).Info("Deleting namespace " + curator.Namespace)
+		err := utils.DeleteClusterNamespace(r.Kubeset, curator.Namespace)
```

Comment:
why do we need this one if managedcluster-import-controller can delete the cluster namespace?

Reply:
Agree. we don't need this.

Reply:
ManagedCluster was not deleting the namespace, because the POST-Ansible jobs were running two long. And I was able to observe in the registration controller, that the AnsibleJobs were detected and the namespace was not removed.  Therefore I added this curator step to also make sure it gets cleaned up.   The Registration controller stops trying after a while.  FYI @leena-jawale @hanqiuzh